docs/quickstart.md

```diff
@@ -95,18 +90,19 @@ npm install firebase reactfire@canary
    ```jsx
    //...
    function Burrito() {
-     // create a ref
-     const firebaseApp = useFirebaseApp();
-     const burritoRef = firebaseApp
-       .firestore()
+     // lazy load the Firestore SDK
+     const firestore = useFirestore();
```

Comment:
@davideast @jamesdaniels How does this pattern look for lazy loading? It throws a Promise until the SDK is loaded

Reply:
LGTM, do you want to provide `use*` options for the other components, even if we aren't utilizing them internally yet. e.g, `useFunctions()`, `useAnalytics()`

Reply:
yeah, I think that's a good call. will do

Reply:
done